feconf.py

```diff
@@ -315,6 +315,9 @@
     'svg': ['svg']
 }
 
+# An array containing the image formats that can be compressed.
+COMPRESSIBLE_IMAGE_FORMATS = ['jpeg', 'png', 'gif']
```

Comment:
Maybe change the *keys* in the constant dict above to named constants: IMAGE_FORMAT_JPEG etc. and use those here. Want to make sure these two sets of strings match each other.

(Leave the values as-is -- those are user-supplied strings.)

Reply:
Done.